model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -89,11 +89,7 @@ data class AnalyzerResult(
      * True if there were any issues during the analysis, false otherwise.
      */
     @Suppress("UNUSED") // Not used in code, but shall be serialized.
-    val hasIssues by lazy {
-        issues.any { it.value.isNotEmpty() } || projects.any { project ->
-            project.scopes.any { scope -> scope.dependencies.any { pkg -> pkg.hasIssues() } }
-        }
-    }
+    val hasIssues by lazy { collectIssues().isNotEmpty() }
```

Comment:
Commit message: Please replace the single-quote after "false" with a back-tick. And could you briefly mention *what* was the issue with the previous code, i.e. what does `collectIssues` do different?

Reply:
Done.

Reply:
Thanks! Sorry, but yet another commit message nit: There's a double space before `collectIssues()`.

Reply:
Done.